TouchEffect.UWP/PlatformTouchEff.cs

```diff
@@ -80,12 +85,24 @@ private void OnPointerCanceled(object sender, PointerRoutedEventArgs e)
         {
             _pressed = false;
             Element.GetTouchEff().HandleTouch(TouchStatus.Canceled);
+            Element.GetTouchEff().HandleHover(HoverStatus.Exited);
+        }
+
+        private void OnPointerCaptureLost(object sender, PointerRoutedEventArgs e)
+        {
+            if(!_intentionalCaptureLoss){
+                _pressed=false;
+                Element.GetTouchEff().HandleTouch(TouchStatus.Canceled);
+                Element.GetTouchEff().HandleHover(HoverStatus.Exited);
+                _intentionalCaptureLoss = false;
+            }
         }
 
         private void OnPointerReleased(object sender, PointerRoutedEventArgs e)
         {
-			_pressed = false;
-			Element.GetTouchEff().HandleTouch(_pressed && (Element.GetTouchEff().HoverStatus == HoverStatus.Entered || !_isHoverSupported) ? TouchStatus.Completed : TouchStatus.Canceled);
+            Element.GetTouchEff().HandleTouch(_pressed && (Element.GetTouchEff().HoverStatus == HoverStatus.Entered || !_isHoverSupported) ? TouchStatus.Completed : TouchStatus.Canceled);
```

Comment:
if(_intentionalCaptureLoss) retrurn;

?

Reply:
I don't understand why this would be necessary here?

Reply:
@piersdeseilligny because of Cancel is already sent by another event handler, isn't it?
We don't want to raise "Cancel" twice

Reply:
Except that if `OnPointerReleased` is called, then `_intentionalCaptureLoss` is set to true, and `OnPointerCaptureLost` doesn't send Cancel; it's one or the other.

However, Cancel is sent twice if leaving the pointer while pressed and releasing it outside of the control (by OnPointerExited, and then by OnPointerReleased)

Reply:
that's a bug as well I believe :)

Reply:
Which is now fixed :) While we're editing that code, I was wondering: what's the purpose of `_isHoverSupported`? Because UWP always fires the PointerEntered and PointerExited events, even on touch devices, so "hover" is technically always supported?

Reply:
@piersdeseilligny ah, really?
I thought about windows 10 phone devices

Reply:
or touch screens

I am not familiar with UWP

Reply:
![image](https://user-images.githubusercontent.com/42496508/69158602-9e1dec00-0ade-11ea-9903-6068a2de2ab4.png)
 
Yep :) Shall I remove it?

Reply:
i think yes)